extensions/odoo_theme/static/style.scss

```diff
@@ -948,140 +902,95 @@ header {
                 font-weight: $fw_bold;
 
                 &:before {
-                    @include o-inline-icon($i-info, 0 .2rem 0 0, baseline, $font-size-base, inherit);
+                    @include o-inline-icon($i-info, 0 $o-icon-margin 0 0, baseline, $font-size-base, inherit);
                     @include media-breakpoint-down(md) {
                         display:none;
                     }
                 }
             }
 
-            &.alert-info, &[class^="admonition"] {
-                border-color: tint-color($brand-info, 30%);
-                background-color: tint-color($brand-info, 85%);
-                color: shade-color($brand-info, 35%);
-
-                > .alert-title, > h3 {
-                    color: $brand-info;
-                    &:before {
-                        content: '#{$i-info}';
-                    }
-                }
+            // Custom Admonitions
+            &[class^="admonition"] {
+                @extend .alert-info;
 
                 code, div[class^="highlight"] {
-                    border-color: tint-color($brand-info, 30%);
+                    border-color: tint-color($info, 30%);
                 }
             }
 
-            &.alert-note {
-                border-color: shade-color($info, 30%);
-                background-color: $gray-lightest;
-                color: shade-color($info, 55%);
-
-                > .alert-title, > h3 {
-                    color: shade-color($info, 45%);
-                    &:before {
-                        content: '#{$i-note}';
-                    }
-                }
-            }
-
-            &.alert-tip, &.tip {
-                border-color: shade-color($color-alpha, 10%);
-                background-color: $gray-lightest;
-                color: shade-color($color-alpha, 55%);
-
-                > .alert-title, > h3 {
-                    color: shade-color($color-alpha, 40%);
-                    &:before {
-                        content: '#{$i-lightbulb}';
-                        @include font-size(1.2rem);
-                    }
-                }
-            }
-
-            &.alert-warning, &.warning {
-                border-color: $brand-warning;
-
-                > .alert-title, > h3 {
-                    color: shade-color($brand-warning, 10%);
-                    &:before {
-                        content: '#{$i-warning}';
-                    }
+            // Note
+            &.alert-primary {
+                > .alert-title:before, > h3:before {
+                    content: '#{$i-note}';
```

Comment:
you can define css variables in the main selector and then apply the customization by class (better inside a loop)... something like:

```scss
.alert {
    > .alert-title:before, > h3:before {
        content: var(--o-alert-icon, #{$i-default-icon});
    }
    
    // [.. smthg similar for borders]
    
    &.alert-success {
        --o-alert-icon: #{$i-whatever};
        --o-alert-border-color: #{$o-color-xhatever}
    }
}



```